client/src/main/java/com/alibaba/nacos/client/config/http/ServerHttpAgent.java

```diff
@@ -413,22 +420,29 @@ private static String getSTSResponse() throws IOException {
 
     @Override
     public String getName() {
-        return serverListMgr.getName();
+        return this.serverListMgr.getName();
     }
 
     @Override
     public String getNamespace() {
-        return serverListMgr.getNamespace();
+        return this.serverListMgr.getNamespace();
     }
 
     @Override
     public String getTenant() {
-        return serverListMgr.getTenant();
+        return this.serverListMgr.getTenant();
     }
 
     @Override
     public String getEncode() {
-        return encode;
+        return this.encode;
+    }
+
+    @Override
+    public void shutdown() throws NacosException{
+        LOGGER.info("do shutdown begin");
```

Comment:
Better add the class name to see which Object is currently closed

Reply:
Okay, I will adjust this codes.